### PR TITLE
RDoc-1912_IndexHistoryView

### PR DESCRIPTION
**Pull request by reebhub**
Documentation of Studio's Index History View

### CODE REVIEW

File: Documentation/5.2/Raven.Documentation.Pages/studio/database/Indexes/index-history-view.markdown [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+---
+
+{NOTE: }
+
+* The **Index History View** allows you to browse the history of changes made 
+  in indexes over time. You can load any past version of an index to reinstate 
+  it as the current active index.  
+
+* A past index version can be reinstated along with its [field options](../../../studio/database/indexes/create-map-index#index-field-options) 
+  and any other settings.
```
[the remark this answers is not in the document]
Comment (reebhub) [reply Documentation/5.2/Raven.Documentation.Pages/studio/database/Indexes/index-history-view.markdown:11]: done

File: Documentation/5.2/Raven.Documentation.Pages/studio/database/Indexes/index-history-view.markdown [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+{NOTE: }
+
+* The **Index History View** allows you to browse the history of changes made 
+  in indexes over time. You can load any past version of an index to reinstate 
+  it as the current active index.  
+
+* A past index version can be reinstated along with its [field options](../../../studio/database/indexes/create-map-index#index-field-options) 
+  and any other settings.  
+  Alternatively, you can load only the version's Map and Reduce sections and leave 
+  other settings of the currently active index unchanged.
```
[the remark this answers is not in the document]
Comment (reebhub) [reply Documentation/5.2/Raven.Documentation.Pages/studio/database/Indexes/index-history-view.markdown:13]: done

File: Documentation/5.2/Raven.Documentation.Pages/studio/database/Indexes/index-history.markdown [hunk 3]
```diff
@@ -0,0 +1,111 @@
+﻿# Index History
+---
+
+{NOTE: }
+
+* You can open an **Index History** dialog for any index, browse the history of 
+  changes made in this index over time, and revert the currently active index to any 
+  of its past revisions.
```
Comment (Danielle9897) [inline Documentation/5.2/Raven.Documentation.Pages/studio/database/Indexes/index-history.markdown:8]: Replace above with:

* An index revision is created for any change made to the index definition.
* By default 10 index revisions are kept. This number can be modified using the Database Settings View
or through configuration (add a link ... /server/configuration/configuration-options)
* The Index History Dialog allows you to browse these revisions and revert the current index definition to any of its past revisions.

File: Documentation/5.2/Raven.Documentation.Pages/studio/database/Indexes/index-history.markdown [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+  changes made in this index over time, and revert the currently active index to any 
+  of its past revisions.  
+
+* When an index is reverted to one of its past revisions, the revision can be loaded 
+  along with its index [field options](../../../studio/database/indexes/create-map-index#index-field-options) 
+  and all other settings.  
+  Alternatively, you can revert only the index's Map and Reduce sections to those of 
+  the selected revision and leave its other settings unchanged.  
+
+* The number of revisions RavenDB keeps for each index can be updated.
```
Comment (Danielle9897) [inline Documentation/5.2/Raven.Documentation.Pages/studio/database/Indexes/index-history.markdown:16]: remove

File: Documentation/5.2/Raven.Documentation.Pages/studio/database/Indexes/index-history.markdown [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+
+* You can open an **Index History** dialog for any index, browse the history of 
+  changes made in this index over time, and revert the currently active index to any 
+  of its past revisions.  
+
+* When an index is reverted to one of its past revisions, the revision can be loaded 
+  along with its index [field options](../../../studio/database/indexes/create-map-index#index-field-options) 
+  and all other settings.  
+  Alternatively, you can revert only the index's Map and Reduce sections to those of 
+  the selected revision and leave its other settings unchanged.
```
Comment (Danielle9897) [inline Documentation/5.2/Raven.Documentation.Pages/studio/database/Indexes/index-history.markdown:14]: Make simpler, replace:
=>
You choose to revert only the 'Map and Reduce' methods or the full index definition.

File: Documentation/5.2/Raven.Documentation.Pages/studio/database/Indexes/index-history.markdown [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+2. **Load Map & Reduce Only**  
+   Click the arrow and choose the "Load Map & Reduce Only" dropdown to load the selected 
+   revision's *Map and Reduce sections* only, leaving its other settings unchanged.  
+
+{PANEL/}
+
+{PANEL: View and Update Number of Index Revisions}
+
+To find how many index revisions RavenDB keeps, and optionally 
+change this number, use the *Database Settings* view.
```
Comment (Danielle9897) [inline Documentation/5.2/Raven.Documentation.Pages/studio/database/Indexes/index-history.markdown:82]: =>
The number of index revisions kept by the server can be modified using the Database Settings View
or through configuration (add a link ... /server/configuration/configuration-options)

File: Documentation/5.2/Raven.Documentation.Pages/studio/database/Indexes/indexes-list-view.markdown [hunk 7]
```diff
@@ -0,0 +1,161 @@
...
+  This index will start processing all relevant documents.  
+
+* The 'old index' (label **'1'** in above figure) will answer queries and index new documents until  
+  the 'new index' (label **'2'** in above figure) is caught up and has indexed everything.  
+
+* Once the 'new index' has completed its work, it will automatically replace the old one.  
+  You can also force an immediate replacement if needed, by clicking **'Swap Now'** (label **'3'** in above figure).  
+
+* RavenDB keeps a [history of index revisions](../../../studio/database/indexes/index-history), 
+  allowing you to revert an index to any of its past revisions.
```
Comment (Danielle9897) [inline Documentation/5.2/Raven.Documentation.Pages/studio/database/Indexes/indexes-list-view.markdown:146]: Add:
=> 
The 'old index' definition can still be referenced as RavenDB keeps a history of index revisions, allowing you to revert an index to any of its past revisions.

File: Documentation/5.2/Raven.Documentation.Pages/studio/database/Indexes/index-history.markdown [hunk 8]
```diff
@@ -0,0 +1,108 @@
+﻿# Index History
+---
+
+{NOTE: }
+
+* An index revision is created for any change made to the index definition.  
+
+* By default 10 index revisions are kept. This number can be modified using 
+  the Database Settings view or through [configuration](../../../server/configuration/configuration-options).
```
Comment (Danielle9897) [inline Documentation/5.2/Raven.Documentation.Pages/studio/database/Indexes/index-history.markdown:9]: Can add a link to 'Database Settings View'

**Review by Danielle9897 — APPROVED** [review]

**Pull request closed without merging**